host/http.go

```diff
@@ -47,6 +47,8 @@ type Host struct {
 
 	listener net.Listener
 
+	maxJobConcurrency uint64
```

Comment:
I thought so too but uint64 works too and is slightly nicer in that it it alleviates the need to further validate for positive input.